### PR TITLE
Add normalize arg to do_dist

### PR DESCRIPTION
**Pull request by hideaki**
### Description
Add normalize arg to normalize measures (key) to do_dist for skv case.

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: tests/testthat/test_pairwise.R [hunk 1]
```diff
@@ -163,6 +163,22 @@ test_that("test do_dist with cmd_scale", {
...
+  test_df <- data.frame(
+    row=rep(paste("row", seq(4)), each=6),
+    col=rep(paste("col", seq(6)), 4) ,
+    val=c(seq(7),seq(7),seq(10))
+  )
+  result_kv <- test_df %>%
+    do_dist(skv = c("row", "col", "val"), diag=TRUE, cmdscale_k = 3, normalize=TRUE)
+
+  expect_equal(ncol(result_kv), 4)
+  expect_equal(result_kv[[2]], result_kv[[2]])
```
Comment (kei51e) [inline tests/testthat/test_pairwise.R]: What is this testing? It should be equal all the time.

Comment (hideaki) [reply tests/testthat/test_pairwise.R]: Right.. Added expected values to the test.

File: tests/testthat/test_pairwise.R [hunk 2]
```diff
@@ -163,6 +163,20 @@ test_that("test do_dist with cmd_scale", {
   expect_equal(result_kv[[4]], result_kv[[4]])
```
Comment (kei51e) [inline tests/testthat/test_pairwise.R]: This one should be updated too. And it should have some comments on the expected values like 'this is normalized', 'this is not normalized'.

Comment (hideaki) [reply tests/testthat/test_pairwise.R]: Added expected value.
About normalized or not, first argument of test_that() indicates that.
```
test_that("test do_dist with cmd_scale with normalize", {
```

**Pull request merged**